src/views/LoanDetails/LoanDetails.js

```diff
@@ -484,10 +506,13 @@ class LoanDetails extends React.Component {
                   value={requestQueueValue}
                 />
               </Col>
-              <Col xs={2}>
+              <Col
+                data-test-loan-actions-history-lost
+                xs={2}
+              >
                 <KeyValue
                   label={<FormattedMessage id="ui-users.loans.details.lost" />}
-                  value="TODO"
+                  value={lostDate ? (<FormattedTime value={lostDate} day="numeric" month="numeric" year="numeric" />) : '-'}
```

Comment:
Please use NoValue component instead of a hyphen for accessibility reasons.

https://github.com/folio-org/stripes-components/pull/1138/files#diff-62058968184332c155544f3fae66caf6R7